corehq/apps/app_manager/static/app_manager/js/modules/module_view.js

```diff
@@ -1,5 +1,7 @@
-hqDefine("app_manager/js/modules/module_view", function () {
+hqDefine("app_manager/js/modules/module_view", ['select2/dist/js/select2.full.min'], function () {
```

Comment:
This is the right thing to do, but unfortunately app manager isn't migrated to RequireJS yet, so you can't add it yet. This is what's causing `test_requirejs_disallows_hqimport` to fail.